src/dotnet-proj-info/Inspect.fs

```diff
@@ -338,10 +355,158 @@ let getProjectInfos log msbuildExec getters additionalArgs projPath =
     |> Result.map (fun _ -> parsers |> List.map (fun parse -> parse ()))
 
 let getProjectInfo log msbuildExec getArgs additionalArgs projPath =
-
+    //TODO refactor to use getProjectInfos
     let template, args, parse =  getArgs ()
 
     projPath
     |> install_target_file log [template]
     |> Result.bind (fun _ -> msbuildExec projPath (args @ additionalArgs))
     |> Result.bind (fun _ -> parse ())
+
+
+
+let getFscArgsOldSdk propsToFscArgs () =
+
+    //from https://github.com/Microsoft/visualfsharp/blob/e5762fd1ed1fcb3fc70c978688204a35cfae1a3e/src/fsharp/FSharp.Build/Microsoft.FSharp.Targets#L264-L309
+    //as string, yes, for easier maintenance to copy/paste from previous source
+    let props =
+        """
+              BaseAddress="$(BaseAddress)"
+              CodePage="$(CodePage)"
```

Comment:
TODO import `Microsoft.FSharp.Targets` with `paket` and load as xml to read the Task attributes. easier to maintain